homeassistant/components/light/mqtt.py

```diff
@@ -37,6 +37,8 @@
 CONF_COLOR_TEMP_COMMAND_TOPIC = 'color_temp_command_topic'
 CONF_COLOR_TEMP_STATE_TOPIC = 'color_temp_state_topic'
 CONF_COLOR_TEMP_VALUE_TEMPLATE = 'color_temp_value_template'
+CONF_COLOR_TEMP_MIREDS_MIN = 'color_temp_mireds_min'
+CONF_COLOR_TEMP_MIREDS_MAX = 'color_temp_mireds_max'
```

Comment:
I think we should call these options `color_temp_min_mireds` and `color_temp_max_mireds` as those are the names used throughout Home Assistant AFAIK.

Reply:
I'll fix it. Thanks.

Reply:
I'll fix it. Thanks.